Add files via upload (#2289)
Codecademy/docs: content/php/concepts/math-functions/terms/acos/acos.md

---

> Title: 'floor()'

Reviewer: ```suggestion
Title: 'acos()'
```

> Description: 'Returns a rounded number down to the nearest integer.'

Reviewer: ```suggestion
Description: 'Returns the angle whose cosine value is equal to the specified value.'

```

Subjects:
  - 'Computer Science'
  - 'Web Design'
  - 'Web Development'
  - 'Math Functions'
Tags:
  - 'Trigonometric'
  - 'Functions'
CatalogContent:
  - 'learn-php'
  - 'paths/computer-science'
---

> 

Reviewer: ```suggestion
```
Remove extra line break.

The built-in math function **`acos()`** determines the `arc cosine` of a given value. The inverse cosine function, which is used to identify the angle whose cosine value is equal to the specified value, is referred to as the `arc cosine` function.

## Syntax

```pseudo
$result = acos(number);
```

> Where `number` is the number that should be Specifies in range -1 to 1.

Reviewer: ```suggestion
Where `number` is the number that the function returns the arc cosine for. Should be specified in range -1 to 1.

```

## Example

```php

> acos(0.64)

Reviewer: The example must be full PHP code. It needs the `<?php ... ?>` tags and something line an `echo()` function to do something with the result of the function.

> // Output: 0.87629806116834

Reviewer: ```suggestion
```
Remove the comment, as this is handled in the description.

```

> The result will be a float value of `0.87629806116834` as the function angle in `rad`.

Reviewer: ```suggestion
The result will be a float value of `0.87629806116834`.
```

> acos(-1) returns the value of Pi.

Reviewer: ```suggestion
`acos(-1)` returns the value of Pi.

```

---
Title: 'acos()'
Description: 'Returns the angle whose cosine value is equal to the specified value.'

Subjects:
  - 'Computer Science'
  - 'Web Design'
  - 'Web Development'
  - 'Math Functions'
Tags:
  - 'Trigonometric'
  - 'Functions'
CatalogContent:
  - 'learn-php'
  - 'paths/computer-science'
---

The built-in math function **`acos()`** determines the `arc cosine` of a given value. The inverse cosine function, which is used to identify the angle whose cosine value is equal to the specified value, is referred to as the `arc cosine` function.

## Syntax

```pseudo
$result = acos(number);
```

Where `number` is the number that the function returns the arc cosine for. Should be specified in range -1 to 1.

## Example

```php

> acos(0.64)

Reviewer: You still need to add the `<?php ... ?>` tags to this example.

---
Title: 'acos()'
Description: 'Returns the angle whose cosine value is equal to the specified value.'

Subjects:
  - 'Computer Science'
  - 'Web Design'
  - 'Web Development'
  - 'Math Functions'
Tags:
  - 'Trigonometric'
  - 'Functions'
CatalogContent:
  - 'learn-php'
  - 'paths/computer-science'
---

The built-in math function **`acos()`** determines the `arc cosine` of a given value. The inverse cosine function, which is used to identify the angle whose cosine value is equal to the specified value, is referred to as the `arc cosine` function.

## Syntax

```pseudo
$result = acos(number);
```

Where `number` is the number that the function returns the arc cosine for. Should be specified in range -1 to 1.

## Example

```php
<?php

> acos(0.64)

Reviewer: ```suggestion
  echo(acos(0.64));
```
So we do something with the result of the function.
